### ISSUE TITLE
Row-Object mapping

### ISSUE THREAD

**Issue by b0bi79** [ClosedXML/ClosedXML#1692]
Hi guys.
I just have one question. To implement a new feature in ClosedXML.Report, I need to know on the basis of which object the row is built. I looked in ClosedXML for the ability to assign an some object to a row or cell, but did not find such feature. Could you please suggest some way to perform STRING-OBJECT mapping? But keep in mind that rows are copied from one place to another and also between sheets.

**Comment by igitur** [comment]
I don't think anything supports this currently. If you had to do it with Excel only, how would you do it?

**Comment by b0bi79** [comment]
Ok, thanks. Then I will develop some kind of binding manager.

**Comment by igitur** [comment]
Wait, I was just checking first whether it's possible in Excel.

If not, we can add an `object IXLCell.MetaData` property for you to store any kind of data in it. But this will not be persisted if you save the file.

**Comment by A9G-Data-Droid** [comment]
I am looking for the opposite of `InsertData` something like `ExtractData` where you can use a POCO, end to end. So you can dump a collection of object into Excel and then extract that same object back out later. 

Are we talking about the same thing in this feature request?

**Comment by jahav** [comment]
@A9G-Data-Droid No, this was basically a question whether a cell/row can hold a reference to an arbitrary c# object. Either directly as a cell value or in some other property. The closest thing to that in the standard is a metadata part and cm(CellMetadataIndex)/vm(ValueMetadataIndex) attributes of a cell.
